### ISSUE TITLE
Support for input_number 

### ISSUE THREAD

**Issue by mattieha** [mattieha/slider-button-card#7]

**Comment by apivaral** [comment]
Yes please! This component is awesome! But it really needs support for input_number please, Thanks!

**Comment by scout1256** [comment]
me2!!

**Comment by djsheetu** [comment]
Would be really cool to be able to use the component with input_number.

**Comment by w1tw0lf** [comment]
Would also like to have this available.

**Comment by meeDamian** [comment]
Is there any technical reason why this isn't supported yet, or just nobody got around to implementing/allowing it yet?

**Comment by guevara777** [comment]
Would love to see support for input_number too. Thank you so much for your great work with this card!!!

**Comment by mkjustuk** [comment]
Would also love to see support for input_number.  It would be great for time selection sliders such as hot water boosts, heating boosts and so forth.

**Comment by jack3308** [comment]
Adding my support for this! Would love for input_number to be available!

**Comment by imval** [comment]
Adding my support on this! That would be amazing.

**Comment by jclsn** [comment]
I am trying to look into it. Thing is I have never coded with typescript before. I think for someone fluent in the language, it would be real easy

**Comment by jclsn** [comment]
Oh there is a PR...
